examples/huggingface-transformers/pipelines.py

```diff
@@ -547,22 +544,26 @@ def __call__(self, inputs: Union[str, List[str]], **kwargs):
             special_tokens_mask = tokens.pop("special_tokens_mask")[0]
 
             # Forward
-            entities = self._forward(tokens)[0][0]
-            input_ids = tokens["input_ids"][0]
+            for current_entities in self._forward(tokens)[0]:
+                input_ids = tokens["input_ids"][0]
```

Comment:
should this be [0] or the index of the item in the batch?

Reply:
Good catch! thanks.